db/migrate/20191001154159_create_search_histories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSearchHistories < ActiveRecord::Migration[5.2]
+  def change
+    create_table :search_histories do |t|
+      t.references :user, foreign_key: true, index: true
+      t.string :keyword
```

Comment:
shouldn't it be citext?

Reply:
right now "kishore" and "Kishore" appear as separate results

Reply:
i guess thats ok - the history is a log of queries and not necessarily unique queries

Reply:
I wasn't aware of citext. But now I am.

Reply:
> right now "kishore" and "Kishore" appear as separate results
This shouldn't have happened. Apologies. I will fix this asap